tests/common/config_mod_utils.py

```diff
@@ -0,0 +1,117 @@
+import dataclasses
+import typing
+
+PREEMPTIBLE_NODE_GROUP_NAME = "preemptible-node-group"
+
+
+@dataclasses.dataclass
+class GPUConfig:
+    cloud: str
+    gpu_name: str
+    node_selector: str
+    node_selector_val: str
+    extra_config: dict
+    min_nodes: typing.Optional[int] = 0
+    max_nodes: typing.Optional[int] = 2
+    node_group_name: typing.Optional[str] = "gpu-node"
+    docker_image: typing.Optional[str] = "quay.io/nebari/nebari-jupyterlab-gpu:2023.7.1"
```

Comment:
Can we set the tag to `constants.DEFAULT_NEBARI_IMAGE_TAG`? 

Or would it make sense to use the [`main` or `latest` tag ](https://quay.io/repository/nebari/nebari-jupyterlab-gpu?tab=tags) to ensure everything is working together? This is what we do for the integration tests.

Reply:
Good point, makes sense!